### PR TITLE
quic: lowering quic brokenness  timer

### PR DESCRIPTION
**Pull request by alyssawilk**
turns out we always ran with roughly this override in production

Testing: updated tests
Docs Changes: n/a
Release Notes: n/a
Platform Specific Features: n/a

### CODE REVIEW

**Comment by alyssawilk** [comment]
Adding renjie for correctness checks, then we can throw over to Envoy maintainers

File: test/common/http/http3_status_tracker_impl_test.cc [hunk 1]
```diff
@@ -101,51 +101,25 @@ TEST_F(Http3StatusTrackerImplTest, MarkBrokenWithBackoff) {
...
-  tracker_.markHttp3Broken();
-  timer_->invokeCallback();
-
+/*
   // Broken period no longer increases.
-  EXPECT_CALL(*timer_, enableTimer(std::chrono::milliseconds(1280 * 60 * 1000), nullptr));
+  EXPECT_CALL(*timer_, enableTimer(std::chrono::milliseconds(1 * static_cast<int>(pow(2, 17)) * 1000), nullptr));
   tracker_.markHttp3Broken();
-  timer_->invokeCallback();
+  timer_->invokeCallback();*/
```
Comment (RenjieTang) [inline test/common/http/http3_status_tracker_impl_test.cc:117]: This block should probably be uncommented?

**Review by RenjieTang — APPROVED** [review]

**Comment by alyssawilk** [comment]
Only windows build broken - I'll merge over it.

**Review by RyanTheOptimist — APPROVED** [review]
Thanks for doing this.

**Pull request merged**